final_project.py

```
@@ -0,0 +1,196 @@
+
+from sklearn.model_selection import train_test_split
+from sklearn.model_selection import cross_val_score
+from sklearn.model_selection import KFold
+from sklearn.metrics import accuracy_score
+from sklearn.preprocessing import MinMaxScaler
+sc = MinMaxScaler()
+import warnings
+warnings.filterwarnings('ignore')
+
+df2 = df1.copy()
+df2
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)